doc/api/fs.md

```diff
@@ -4719,6 +4573,92 @@ The following constants are meant for use with the [`fs.Stats`][] object's
   </tr>
 </table>
 
+## File System Flags
+
+The following flags are available wherever the `flag` option takes a
+string:
+
+* `'a'` - Open file for appending.
+The file is created if it does not exist.
+
+* `'ax'` - Like `'a'` but fails if `path` exists.
```

Comment:
`` `path` `` -> `` path ``?

Reply:
All ``path`` -> `the path` resolved in 14da387a73a827670e5c379da727bbb489b93e3b